site-builder/src/main.rs

```diff
@@ -206,73 +209,40 @@ mod default {
     }
 }
 
-/// Returns the path if it is `Some` or any of the default paths if they exist (attempt in order).
-pub fn path_or_defaults_if_exist() -> Result<PathBuf, anyhow::Error> {
-    // Construct the default paths. Warning! The order is important.
-    let sites_config_name = "sites-config.yaml";
-    let mut defaults: Vec<PathBuf> = vec![];
-    if let Ok(xdg_config_home) = std::env::var("XDG_CONFIG_HOME") {
-        // $XDG_CONFIG_HOME/walrus/sites-config.yaml
-        let xdg_config_home = PathBuf::from(xdg_config_home)
-            .join("walrus")
-            .join("sites-config.yaml");
-        if xdg_config_home.exists() {
-            return Ok(xdg_config_home);
-        }
-        defaults.push(xdg_config_home);
-    }
-    // Define the home directory. If it doesn't exist, use `/`.
-    let home_dir = match home::home_dir() {
-        Some(dir) => dir,
-        None => {
-            return Err(anyhow::anyhow!(
-                "failed to get home directory. $HOME needs to be defined."
-            ));
-        }
+/// Returns the default paths for the sites-config.yaml file.
+pub fn sites_config_default_paths() -> Vec<PathBuf> {
+    let mut default_paths = vec![SITES_CONFIG_NAME.into()];
```

Comment:
praise: nice idea to add every path to the same vec (including the default argument of the config)